<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CExtensions</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CExtensions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Video Extensions:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="3">
       <widget class="QListWidget" name="videoExtensions">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="addVideoExt">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/add.png</normaloff>:/SABUtilsResources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="delVideoExt">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/delete.png</normaloff>:/SABUtilsResources/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="recomputeBtn">
          <property name="text">
           <string>Re-compute Defaults</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Subtitle Extensions:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QToolButton" name="addSubExt">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/add.png</normaloff>:/SABUtilsResources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="delSubExt">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/delete.png</normaloff>:/SABUtilsResources/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QListWidget" name="subtitleExtensions">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>227</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
